README.md

```diff
@@ -333,3 +341,15 @@ lsr_some_shell_func ....
   etc.
 * `LSR_TOX_ENV_DIR` - the full path to the directory for the test environment
   e.g. `/path/to/ROLENAME/.tox/env-3.8`
+
+### Working on collections docs
+
+It can be tricky to get the plugin and module docs to render correctly.  Use a
+workflow like this:
+```
+> LSR_ANSIBLE_TEST_DEBUG=true LSR_ANSIBLE_TEST_TESTS=ansible-doc \
+  tox -e collection,ansible-test
```

Comment:
May I ask what happens if we run just `tox -e ansible-test` or `tox -e ansible-test,collection`? It could finish with the `Collection format only.` message? If that's the case, it might be helpful what to do in the error message?

Reply:
> May I ask what happens if we run just `tox -e ansible-test` or `tox -e ansible-test,collection`?

You will get an error like this: `runansible-test.sh: There is no collection at .tox/ansible_collections/fedora/linux_system_roles`

> It could finish with the `Collection format only.` message? If that's the case, it might be helpful what to do in the error message?

ok - I can make the message something like `runansible-test.sh: There is no collection at .tox/ansible_collections/fedora/linux_system_roles - please run 'tox -e collection' first`